### DISCUSSION TITLE
Let's try discussions

### DISCUSSION THREAD

**Discussion by vroland** [vroland/epdiy#288] [Announcements]
## 👋 Let's try to discussions

Hi,
so far we had a slack which was originally intended for helping each other, sharing projects, discussing design decisions, etc. However, it doesn't quite fit our needs at this point: The member count is capped when not paying a significant amount of money, the invite link expires regularly, and in the free version access to chat history is limited. That way it is of limited use for the community as not everyone has access to it. And frankly, I haven't been reading it a lot lately. 
So let's try to discuss right here on GitHub and see how it goes :)

**Comment by martinberlin** [comment]
Great, thanks for opening this section. I already announced in Slack that we are moving slowly here.
Let’s see if that works. 
I’m working in an LVGL update to version 8.3 and it’s already stable:
https://github.com/martinberlin/lv_port_esp32-epaper
I would like to have some help from people testing and also helping to reverse engineer and make drivers for Kindle like panels.

**Comment by schuhumi** [comment]
That's a really great change that I appreciate a lot! I'm not sure why slack (or discord for that matter) is so popular in foss development, while the issues Valentin lists are clearly diametrical to the general foss ethos. Ironically, 99.9% of the code running on their servers is open source (os, drivers, network stack, ...), and the application they maintain and serve is the last 0.1%, like a cherry on a cake. They get the cake for free, but do not want to share a reasonable amount of their cherry even for the people that work on foss cake. Of course there's limits to bandwidth, storage and compute they could reasonably afford for giving back, but stuff like limiting history is just mocking the very movement they profit from...

**Comment by aceisace** [comment]
I would strongly suggest switching to Discord @vroland . I can even help with setting up a webhook-bot and some triggers if required. Links don't expire, the history remains and file-sizes up to 25MB are allowed. Slack is just too limited for this kind of thing and getting funding for slack doesn't make much sense if there already is a free version, i.e. Discord. As the project will grow further, managing issues will get more difficult too. Plus, it's easier to get quick help from existing users on Discord than on Github issues

**Reply by tobymurray** [reply]
If GitHub Discussions aren't a good fit, I think [Discourse](https://www.discourse.org/) is generally more aligned with the goals shared in this discussion?

**Comment by martinberlin** [comment]
I’m not a big fan of Discord. You can of course start something there and it’s great if it works I know that there are nice maker communities there.
But the question is that many of us don’t have extra time every week to hang in another social network.
And that’s why Discussion topics are fine here so Valentin can just log only in GitHub and read this in one place.
I dislike Discord and all the pop ups colors and stuff that bothers my eyes. But if you guys enjoy it please go ahead.

**Comment by vroland** [comment]
I agree, I feel like having the discussions here is a bit easier for everyone, and the place I would frequent most. But I'm not opposed to people also using discord, I'd just not make an effort to switch over to it.

**Reply by martinberlin** [reply]
I also fully agree with [Shawn Hymel](https://twitter.com/ShawnHymel/status/1789318464403832947) in this point. The great point of Discussions is that is also scanned by google. Anyone searching on "how to solve this problem with epdiy" will also get, in case it exists, the Discussions result link to research. This does not happen in Discord, since only members can check what is inside, that's why I think this is a nice solution.
